salt/metalk8s/repo/redhat.sls

```diff
@@ -61,6 +61,7 @@ Configure {{ repo_name }} repository:
   {%- endif %}
     - repo_gpg_check: {{ repo_config.repo_gpg_check }}
     - enabled: {{ repo_config.enabled }}
+    - proxy: _none_
```

Comment:
Me too, i couldn't figure it upon initial review then I stumbled on this  https://github.com/saltstack/salt/issues/35760

Reply:
It sets the entry `proxy=_none_` in each repository configuration file as it is explained in `man yumconf`:

**proxy** URL to the proxy server for this repository. Set to '\_none\_' to disable the global proxy setting
for this repository. If this is unset it inherits it from the global setting

The goal is to handle the case when the user has a yum or system-wide defined proxy.

It's not in the doc, but you can see in the code that almost any kwargs is a valid parameter (except salt vars & those starting with `__`).

Reply:
If two people (and even one person in fact) didn't understand the meaning of this, that's a clear indication that this deserves a comment :wink: 
Especially since it's not documented.

Reply:
Though it was pretty clear, but I'm used to yum syntax, maybe it's not.
I've added a comment.